iOS/Flows/Conversation/_New/ConversationViewControllerDataSource.swift

```diff
@@ -100,25 +95,32 @@ class ConversationCollectionViewDataSource: CollectionViewDataSource<Conversatio
                 snapshot.insertItems([.message(message.id)],
                                      in: sectionID,
                                      atIndex: index.item)
-                scrollToLatestMessage = true
+                if message.isFromCurrentUser {
+                    scrollToLatestMessage = true
+                }
             case .move:
                 snapshot.deleteItems(snapshot.itemIdentifiers(inSection: sectionID))
                 snapshot.appendItems(conversationController.messages.asConversationCollectionItems,
                                      toSection: sectionID)
             case .update(let message, _):
-                snapshot.reloadItems([message.asConversationCollectionItem])
+                snapshot.reconfigureItems([message.asConversationCollectionItem])
```

Comment:
Changing from reload to reconfigure makes this look A LOT better. The cells don't flicker and everything moves much smoother.